src/generated/resources/data/tconstruct/tinkering/modifiers/melee_protection.json

```diff
@@ -6,8 +6,7 @@
       "type": "tconstruct:max_armor_stat",
       "allow_broken": false,
       "each_level": 0.005,
-      "key": "tconstruct:use_item_speed",
-      "key2": "tconstruct:melee_protection"
+      "stat_key": "tconstruct:use_item_speed"
```

Comment:
I originally made the comment about the keys back when you had 2 keys just differentiated by numbers. Though I do wonder whether `stat_key` is not better than `key` which I used in `ArmorStatModule`. Not sure whether consistency or readability is more important here.

Reply:
I chose to use key